Source/SwiftLintFramework/Rules/VerticalWhitespaceRule.swift

```diff
@@ -93,22 +87,16 @@ public struct VerticalWhitespaceRule: CorrectableRule,
             blankLinesSections.append(lineSection)
         }
 
-        // matching all accurrences of /* */
-        let matchMultilineComments = "/\\*(.|[\\r\\n])*?\\*/"
-        let comments = file.matchPattern(matchMultilineComments)
-
+        // filtering out violations in comments and strings
+        let stringAndComments = Set(SyntaxKind.commentAndStringKinds())
+        let syntaxMap = SyntaxMap(file: file)
         var result = [(lastLine: Line, linesToRemove: Int)]()
         for eachSection in blankLinesSections {
             guard let lastLine = eachSection.last else { continue }
-
-            // filtering out violations within a multiple comment block
-            let isSectionInComment = !comments.filter {
-                (eachRange, _ ) in eachRange.intersectsRange(lastLine.range)
-            }.isEmpty
-
-            if isSectionInComment {
-                continue  // skipping the lines found in multiline comment
-            } else {
+            let kindInSection = syntaxMap.tokensIn(lastLine.byteRange)
+                                                        .map { $0.type }
```

Comment:
instead of mapping twice, how about `.flatMap { SyntaxKind(rawValue: $0.type) }`?